jetstack/tarmak
# WIP: Add better acceptance tests for fluentbit

This PR improves the acceptance tests for fluent_bit

Goals:
- Test version is the expected one
- Ensure  aws-es-proxy is setup correctly and working
- Update of configuration with and without AWS proxy

```release-note
NONE
```
